### PR TITLE
SfsClient subtree update for proxy support

### PR DESCRIPTION
**Pull request by nidietr-MSFT**
- [x] I have signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs).
- [x] This pull request is related to an issue.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-cli/pull/5044)

### CODE REVIEW

File: src/SfsClient/readme.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ## SfsClient
 
-Do not change code under the sfs-client directory; it contains sfs-client source code from commit [be733af](https://github.com/microsoft/sfs-client/commits/be733af).  
+Do not change code under the sfs-client directory; it contains sfs-client source code from commit [c639a50](https://github.com/microsoft/sfs-client/commit/c639a50).
```
Comment (arthuraraujo-msft) [inline src/SfsClient/readme.md:3]: Please wait for the release of the client to happen before picking the commit hash.

Comment (arthuraraujo-msft) [reply src/SfsClient/readme.md:3]: Do use this release https://github.com/microsoft/sfs-client/releases/tag/1.1.0

**Comment by yao-msft** [comment]
For subtree updates, to keep the history, we'll usually do something similar to below
`git subtree add --prefix=src/SfsClient/sfs-client https://github.com/microsoft/sfs-client.git 1.1.0 --squash`
This will create 2 commits. We do not modify these 2 commits. All changes outside of the subtree pull will go to a 3rd commit (readme, etc). Also please update the sfs-client version to be 1.1.0 in [SfsClient.vcxproj](https://github.com/microsoft/winget-cli/blob/master/src/SfsClient/SfsClient.vcxproj) in the 3rd commit.

**Comment by nidietr-MSFT** [comment]
Thanks Yao, Arthur. 

Then, for clarity, let's cancel this PR.

I have now created a new clean PR for SfsClient update in WinGet
https://github.com/microsoft/winget-cli/pull/5052

This new PR:
- directly pulls SfsClient 1.1.0 release, through its 2 unmodified commits from github subtree pull xxx -squash
- has as a separate 3rd commit the readme.md & SfsClient project updated

**Pull request closed without merging**